src/templates/it/2019/base.html

```diff
@@ -0,0 +1,28 @@
+{% extends "%s/base.html" % lang %}
+
+{% block methodology_stat_1 %}5.8M{% endblock %}
+{% block methodology_stat_2 %}20.9 TB{% endblock %}
+{% block methodology_description %}
+Se non diversamente specificato, le metriche in tutti i 20 capitoli del Web Almanac provengono dal dataset dell'archivio HTTP. HTTP Archive è un progetto gestito dalla comunità che tiene traccia del modo in cui è costruito il Web dal 2010. Utilizzando WebPageTest e Lighthouse "under the hood", i metadati di quasi 6 milioni di siti Web vengono testati mensilmente e inclusi in un database BigQuery pubblico per l'analisi. Il set di dati di luglio 2020 è stato utilizzato come base per le metriche del Web Almanac. Per ulteriori informazioni, vedere la pagina Methodology.
```

Comment:
```suggestion
Se non diversamente specificato, le metriche in tutti i 20 capitoli del Web Almanac provengono dal dataset dell'archivio HTTP. HTTP Archive è un progetto gestito dalla comunità che tiene traccia del modo in cui è costruito il Web dal 2010. Utilizzando WebPageTest e Lighthouse "under the hood", i metadati di quasi 6 milioni di siti Web vengono testati mensilmente e inclusi in un database BigQuery pubblico per l'analisi. Il set di dati di Luglio 2019 è stato utilizzato come base per le metriche del Web Almanac. Per ulteriori informazioni, vedere la pagina Methodology.
```

Reply:
Interestingly it was "agosto 2020" (lowercase) in 2020 version which is why I used lowercase here. Should it be capital "A" there? If so I can change that as part of this.

Reply:
A quick Google suggest shouldn't be capitalised:
- https://www.thoughtco.com/italian-capitalization-in-grammar-2011424
- https://www.thinklanguage.com/sites/thinklanguage.com.italian/files/thinkitalian_july2016_grammar.pdf
- https://www.fluentu.com/blog/italian/months-in-italian/

So I say leave as is.

Planning a release now and, as this is the only outstanding question, I say we merge this and make any further changes in a follow up PR.

Reply:
I might have lived in England for too long :D